addons/pos_mercado_pago/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <record id="enable_portal_payment" model="ir.config_parameter" forcecreate="0">
+        <field name="key">pos_mercado_pago.platform_id</field>
+        <field name="value">dev_cdf1cfac242111ef9fdebe8d845d0987</field>
```

Comment:
How sensitive is this information?
If it's a unique identifier which links Odoo to Mercado Pago, shouldn't it be avoided to add it to our open source code / runbot etc.?

Reply:
It's not necessarily sensitive. But it is probably always better to have it be more private than public. I will be talking with other devs here tomorrow to get a recommendation on best practices to secure it properly. Thank you for the reminder, we can also probably regenerate a new key before merging to guarantee that even in commit history the identifier isn't activated.